core/core-db-repositories/src/main/resources/db/migration/postgresql/V1_0_0__Base_schema.sql

```diff
@@ -44,6 +44,18 @@ create table if not exists notification_email (
   template_name varchar(255)
 );
 
+create table if not exists notification_email_file_attachment (
```

Comment:
Please polish the order of columns, the most valuable ones should appear first

Reply:
Changed the order and I've added the same script in mySql setup file, which I missed.